pavelib/assets.py

```diff
@@ -213,12 +312,105 @@ def compile_coffeescript(*files):
 @no_help
 @cmdopts([
     ('system=', 's', 'The system to compile sass for (defaults to all)'),
+    ('themes_dir=', '-td', 'The themes dir containing all themes (defaults to None)'),
+    ('themes=', '-t', 'The theme to compile sass for (defaults to None)'),
     ('debug', 'd', 'Debug mode'),
     ('force', '', 'Force full compilation'),
 ])
 def compile_sass(options):
     """
-    Compile Sass to CSS.
+    Compile Sass to CSS. following is a list of some possible ways to use this command.
+
+    Command:
+        1. paver compile_sass
+    Description:
+        compile sass files for both lms and cms
```

Comment:
@andy-armstrong yes, only those themes are compiled that are mentioned via command line argument. Since, paver commands run outside of django environment so we can not determine all the installed themes (from `Site` model).